sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/SparkMetadataOperationSuite.scala

```diff
@@ -17,63 +17,21 @@
 
 package org.apache.spark.sql.hive.thriftserver
 
-import java.util.{Arrays => JArrays, List => JList, Properties}
-
-import org.apache.hive.jdbc.{HiveConnection, HiveQueryResultSet}
-import org.apache.hive.service.auth.PlainSaslHelper
-import org.apache.thrift.protocol.TBinaryProtocol
-import org.apache.thrift.transport.TSocket
+import java.sql.ResultSet
 
 class SparkMetadataOperationSuite extends HiveThriftJdbcTest {
 
   override def mode: ServerMode.Value = ServerMode.binary
 
   test("Spark's own GetSchemasOperation(SparkGetSchemasOperation)") {
-    def testGetSchemasOperation(
```

Comment:
Help me understand, where does all this deleted logic go? you're saying it just isn't used at all in these tests?

Reply:
Before this pr, we constrctor a HiveConnection, a TCLIService.Client, ... and get `ResultSet` from:
```scala
        rs = new HiveQueryResultSet.Builder(connection)
          .setClient(client)
          .setSessionHandle(sessHandle)
          .setStmtHandle(client.GetSchemas(schemaReq).getOperationHandle)
          .build()
```

After this pr, we get `ResultSet` from `statement.getConnection.getMetaData.getSchemas`

cc @juliuszsompolski